### PR TITLE
Finishes the index and team pages for the website.

### PR DESCRIPTION
**Pull request by juliusiv**
It's huge, sorry.

### CODE REVIEW

File: app/assets/stylesheets/main.less [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+.m8-name, .m8-roles {
+  margin-bottom: 4%;
+}
+
+.m8-roles, .m8-interests {
+  color: @dark-gray;
+}
+
+/* These attributes should be applied when the screen is big.
```
Comment (raymondjacobson) [inline app/assets/stylesheets/main.less:202]: I guess the way these rules are written for big browsers it overwrites the top ones? Maybe two media queries? Dunno. nbd

File: app/assets/stylesheets/mixins.less [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  -o-transform: rotateY(@deg);
+  transform: rotateY(@deg);
+}
+
+.transform-translateY(@percentage) {
+  -webkit-transform: translateY(@percentage);
+  -moz-transform: translateY(@percentage);
+  -o-transform: translateY(@percentage);
+  transform: translateY(@percentage);
+}
```
Comment (raymondjacobson) [inline app/assets/stylesheets/mixins.less]: This +100 dope.

File: app/controllers/Whereuat.scala [hunk 3]
```diff
@@ -70,6 +70,14 @@ class Whereuat extends Controller {
 
 
   // Route actions
+  def index = Action {
+    Ok(views.html.index())
+  }
+
+  def team = Action {
```
Comment (raymondjacobson) [inline app/controllers/Whereuat.scala:77]: I like the idea of having a single page app for this, but good for now. future release

File: app/views/main.scala.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <script type="text/javascript" src='@routes.Assets.versioned("javascripts/index.js")'></script>
+  </head>
+  <body>
+    <div class="container">
+      @content
+      <div class="push"></div>
+    </div>
+
+    <footer>
+      &copy 2016 <span>whereu@@</span>. All rights reserved.
```
Comment (raymondjacobson) [inline app/views/main.scala.html]: need ; after copy

File: app/assets/stylesheets/main.less (hunk 1 above)
Comment (juliusiv) [inline app/assets/stylesheets/main.less:202]: tried to mimic how skeleton does it

File: app/views/team.scala.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+@main(title="whereuat | team") {
+<div class="banner">
+  <img id="ourteam" src='@routes.Assets.versioned("images/ourteam.png")'>
+</div>
+@teammate("Julius Alexander IV",
+          "Backend & Android dev, Scala, Java, Clojure",
```
Comment (raymondjacobson) [inline app/views/team.scala.html:6]: &copy ;

File: app/controllers/Whereuat.scala (hunk 3 above)
Comment (juliusiv) [inline app/controllers/Whereuat.scala:77]: it's just easier this way i think. don't have to rely on angular or something and it's super simple

**Comment by raymondjacobson** [comment]
If you think this is big... see iOS heh...

LGTM % add a new line at the end of each file ;)

**Comment by juliusiv** [comment]
fixed stuff, check it out, will squash everything when lgtm'd

**Pull request merged**